applications/GeoMechanicsApplication/tests/fixed_spatial_variation/simple_mesh.mdpa

```diff
@@ -0,0 +1,104 @@
+Begin Table 1 X WATER_PRESSURE
+    0.0000000000 0.0000000000
+    1.0000000000 -5000
+End Table
+
+Begin Table 2 X WATER_PRESSURE
+    0.0000000000 -40000
+    1.0000000000 -30000
+End Table
+
+Begin Properties 1
+End Properties
+
+
+Begin Nodes
+  1  0.0000000000 -2.0000000000 0.0000000000
+  2  1.0000000000 -2.0000000000 0.0000000000
+  3  1.0000000000 0.0000000000 0.0000000000
+  4  0.0000000000 0.0000000000 0.0000000000
+  5  0.5000000000 -2.000000000 0.0000000000
+  6  1.0000000000 -1.000000000 0.0000000000
+  7  0.5000000000 0.0000000000 0.0000000000
+  8  0.0000000000 -1.000000000 0.0000000000
+End Nodes
+
+
+Begin Elements UPwSmallStrainElement2D8N
+  1  1  1 2 3 4 5 6 7 8
+End Elements
+
+
+
+Begin SubModelPart soil
+  Begin SubModelPartTables
```

Comment:
Not sure why we write all these empty Begin End structures. I think the program functions correctly without them.

Reply:
Works indeed, deleted!